packages/react/src/patterns/sub-patterns/Layout/__stories__/Layout.stories.js

```diff
@@ -56,7 +52,9 @@ storiesOf('Patterns (Sub-Patterns)|Layout', module)
             layoutOptions,
             layoutOptions['layout-06']
           )}
-          stickyOffset={number('Sticky offset (in pixels)', 0)}>
+          stickyOffset={number('Sticky offset (in pixels)', 0)}
+          border={boolean('Optional border:', false)}
+          nested={true}>
```

Comment:
This should not be nested based on my understanding of what you're doing here because you're not nesting it anywhere in the grid. it stands on its own.

Reply:
I removed this on the latest version